TESTING.md

```diff
@@ -0,0 +1,62 @@
+# Testing metaschema-xslt
+
+An important secondary goal of this project is that its processes should be *validable*, which implies transparent and auditable testing processes.
```

Comment:
Minor, NB: Should this be *validatable*?
```suggestion
An important secondary goal of this project is that its processes should be *validatable*, which implies transparent and auditable testing processes.
```

Reply:
Hm, lemme look that up....

I looked it up and (whew) it turns out that 'validable' is indeed [correctly formed](https://english.stackexchange.com/questions/37964/word-for-something-that-can-be-validated), as I thought, from the Latin root -- but admittedly rare.

Yet as so often, a problem that isn't real reveals one that is. The sentence can be reworded:

```suggestion
An important secondary goal of this project is that its processes should be reliable, which implies transparent and auditable testing processes.
```

nb I have avoided the word "demonstrability".